ci-operator/config/openshift/operator-framework-operator-controller/openshift-operator-framework-operator-controller-release-4.20.yaml

```diff
@@ -3,18 +3,10 @@ base_images:
     name: commitchecker
     namespace: ci
     tag: "4.15"
-  ocp_4.18_base-rhel9:
-    name: "4.20"
-    namespace: ocp
-    tag: base-rhel9
   ocp_4.19_base-rhel9:
```

Comment:
`ocp_4.20_base-rhel9` ?

Reply:
Or maybe it doesn't exist?

Reply:
Looking at all of this again (and looking at our Dockerfiles), these mappings confuse me. Do we need any of the mappings? Or are the Dockerfiles in our code repo self-contained wrt their image refs?